iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -2057,4 +2057,32 @@ public void testSnycProperties() throws TException, InterruptedException {
   private String getCurrentSnapshotForHiveCatalogTable(org.apache.iceberg.Table icebergTable) {
     return ((BaseMetastoreTableOperations) ((BaseTable) icebergTable).operations()).currentMetadataLocation();
   }
+
+  @Test
+  public void testCreateTableWithPercentInName() throws IOException {
+    Assume.assumeTrue("This test requires Hive Version 4.", HiveVersion.min(HiveVersion.HIVE_4));
```

Comment:
Please remove this line, I think we don't need this.

Reply:
This check was needed on the Iceberg side to get the test to pass - https://github.com/apache/iceberg/pull/9667

I included it here so the class was consistent across both but I've had other issues with the formatting that required them to be different.  I can remove it.